zebra-consensus/src/router.rs

```diff
@@ -247,21 +238,6 @@ where
     // Give other tasks priority before spawning the download and checkpoint tasks.
```

Comment:
```suggestion
    // Give other tasks priority before spawning the checkpoint task.
```

Reply:
Applied in https://github.com/ZcashFoundation/zebra/pull/7844/commits/8ef372dd22edddf66b9503e81b9e576b628ef657.